src/headless/converse-core.js

```diff
@@ -223,6 +223,7 @@ _converse.default_settings = {
     ],
     message_carbons: true,
     nickname: undefined,
+    omemo_default: false,
```

Comment:
This needs to go into [src/converse-omemo.js](https://github.com/conversejs/converse.js/blob/master/src/converse-omemo.js) and you need to use [_converse.api.settings.update](https://conversejs.org/docs/html/api/-_converse.api.settings.html#.update) inside the plugin's [initialize](https://github.com/conversejs/converse.js/blob/master/src/converse-omemo.js#L240) method.

Other than that, looks good :+1:

Reply:
How's that?